tools/rapids-package-name

```diff
@@ -15,20 +15,27 @@ fi
 pkg_type="$1"
 
 append_cuda=0
-append_pyver=0
 append_wheelname=0
+append_pyver=0
+append_arch=0
 
 case "${pkg_type}" in
   conda_cpp)
     append_cuda=1
+    append_arch=1
     ;;
   conda_python)
     append_cuda=1
     append_pyver=1
+    append_arch=1
     ;;
   wheel_python)
-    append_pyver=1
     append_wheelname=1
+    # Pure wheels do not need a pyver or arch
+    if [[ ! -v RAPIDS_PY_WHEEL_PURE ]] || [[ "${RAPIDS_PY_WHEEL_PURE}" != "1" ]]; then
```

Comment:
Honestly probably more sophisticated handling than we strictly need. I'd also be fine with just the first clause and skipping the second check. Also, did someone request the change from a different argument to an env var?

I don't have strong feelings on either so feel free to merge as-is, just wondering what prompted those changes.

Reply:
This script is not called directly. You have to inject the knowledge of whether the wheel is pure or not when calling the higher level wheel upload/download script. This is the same way that we are injecting the custom wheel name for some packages so I copied the design of using an env var as input.

Reply:
Ah right sorry I got turned around because of how you did this originally.